src/ducks/ui/listevisning-selectors.ts

```diff
@@ -6,6 +6,7 @@ import {
     AAP_YTELSE_MAXTID,
     AAP_YTELSE_UNNTAK,
     I_AVTALT_AKTIVITET,
+    mapFilternavnTilFilterValue,
```

Comment:
Siden vi importerer konstantene direkte for de andre filtrene så kan vi gjøre det samme her, for å være konsistente.